sonar_driver:
        ros__parameters:
                port: "/dev/ttyUSB0"
                topic: "/sonar"
                rate: "10"
                baudrate: 115200
gamepad_driver:
        ros__parameters:
                topic: "/cmd_vel"
                rate: 1

motor_driver:
       ros__parameters:
               topic: "/cmd_vel"
               port: "/dev/ttyUSB0" 

motor_diag:
        ros__parameters:
                topic: "/motor_diag"
                port: "/dev/ttyUSB0"
                rate: "5"
joy_driver:
        ros__parameters:
                topic_vel: "/joy"
                topic_btn: "/btn"
                port: "/dev/ttyUSB0"
                rate: "2020"
gps_driver:
        ros__parameters:
                port: "/dev/ttyUSB0"
                topic: "/fix"
                baudrate: 9600 
